interpreter.py

```diff
@@ -198,6 +236,10 @@ def exec_operation(self, instruction):
             var_val = self.lookup_var(args[0])
             assert isinstance(var_val, Float)
             return Float(var_val.value)
+        elif opcode == LLVMZExt:
+            var_val = self.lookup_var(args[0])
+            assert isinstance(var_val, Integer)
+            return Integer(var_val.value)
```

Comment:
Objects are either immutable or not -- "mostly immutable" is as meaningful as "slightly pregnant" (i.e. not very meaningful!) ;)